user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/update/ExperimentUpdateDialog.java

```diff
@@ -9,18 +9,27 @@
 import com.vaadin.flow.component.html.Span;
 import com.vaadin.flow.component.textfield.TextField;
 import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.provider.SortDirection;
 import java.io.Serial;
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Comparator;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 import life.qbic.datamanager.views.events.UserCancelEvent;
 import life.qbic.datamanager.views.general.DialogWindow;
+import life.qbic.datamanager.views.projects.project.experiments.experiment.create.ExperimentAddDialog;
+import life.qbic.datamanager.views.projects.project.experiments.experiment.create.ExperimentAddDialog.ExperimentDraft;
```

Comment:
Should not use the same data object. We introduce a dependency here where none is required. The dialogs may change for different reasons at different times.